### ISSUE TITLE
Support TP-Link TL-WDR3600 v1

### ISSUE THREAD

**Issue by ShapeShifter499** [attila-lendvai/openwrt-auto-extroot#24]
I have a couple of TP-Link TL-WDR3600 v1 routers. I'm posting here because of a strange quirk I noticed and I'm not sure how to debug for. Hoping someone here has an idea what could be up.

When trying the build generated from this project it seems to work but requires a manual power cycle after the USB storage is setup. 

What I've observed is that the router will turn on and with the below LED code flash the appropriate LEDs showing that the script is setting up along with the USB thumb drive flashing itself. After seeing the router reboot the first time both the router and USB will flash LEDs more signifying that it's installing packages, then a second reboot happens. It's at the second reboot where it seems to hang. The USB LED will not come on but the LEDs for router power and connected Ethernet ports do come on. It seems to get into a odd state where internet is "passing through" like a unmanaged switch but it doesn't bother with the USB or respond to SSH or WebUI requests, it's not accessible over 192.168.1.1. 

If I manually cycle power at this state the router reboots normally and everything appears to work as if nothing is wrong. I can at this point even run a reboot command from he WebUI LUCI.

Any thoughts what might be happening? I'm hazarding a guess that after setup and manually power cycling once that it is good to go. 

LED code
```
    *tl-wdr3600*)
        autoprovisionUSBLed="green:wlan2g"
        autoprovisionStatusLed="green:wlan5g"
        ;;
```

**Comment by ShapeShifter499** [comment]
I'll try swapping the USB later when I can get to it again. But is it possible the issue is just a flaky or poorly made USB thumb drive?

**Comment by attila-lendvai** [comment]
one thing that may matter is soft reboot vs. hard reboot, but you wrote that subsequent LuCI reboots work, so i have no useful idea.

if you can reproduce it on a vanilla OpenWrt, then it should be reported upstream. much more expert attention is gathered there.

**Comment by ShapeShifter499** [comment]
I did a lot of testing with an assortment of USB thumb drives I had on hand. Only a couple gave me issue, mainly the no name USB thumb drives I got as freebies over the years. One such USB device appeared fine for transferring files but sometimes would fail to power up if power cycled a bunch of times in a row in a small time frame.

If anyone experiences issues, I'd try a different USB thumb drive first.

I now have a different USB drive working fine with this model and this project code.
